python/dcr_template.py

```diff
@@ -447,22 +483,23 @@ def _calc_psf_model(self):
         dcr_shift = []
         p0 = self.psf_size//2 - self.kernel_size//2
         p1 = p0 + self.kernel_size
-        for _img, calexp in enumerate(self.exposures):
+        for _img, exp in enumerate(self.exposures):
             el = self.elevation_arr[_img]
             az = self.azimuth_arr[_img]
 
+            # Use the measured PSF as the solution of the shifted PSFs.
+            # Taken at zenith, since we're solving for the shift and don't want to introduce any extra.
             dcr_genZ = _dcr_generator(self.bandpass, pixel_scale=self.pixel_scale, elevation=90., azimuth=az)
+            psf_mat.append(_calc_psf_kernel_full(exp, dcr_genZ, return_matrix=False,
+                           x_size=self.psf_size, y_size=self.psf_size))
+            # Calculate the expected shift (with no psf) due to DCR
```

Comment:
Again...could you expand on this?

Perhaps a nice detailed doc string for this method would be a good place to put an overview of what this method does, in which case the existing code comments might suffice as reminders.